### PR TITLE
Build image based on Alpine so we can install CA certificates

### PR DESCRIPTION
**Pull request by drcapulet**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,7 @@
-FROM scratch
-COPY gcp-artifact-registry-docker-proxy /
-ENTRYPOINT ["/gcp-artifact-registry-docker-proxy"]
+FROM alpine
```
Comment (derektamsen) [inline Dockerfile:1]: Recommend pinning this to a specific version of alpine. `latest` [refers to 3.19.1](https://hub.docker.com/layers/library/alpine/3.19.1/images/sha256-6457d53fb065d6f250e1504b9bc42d5b6c65941d57532c072d929dd0628977d0?context=explore)

```suggestion
FROM alpine:3.19.1
```

**Review by derektamsen — APPROVED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+
+RUN apk --no-cache add curl
```
Comment (derektamsen) [inline Dockerfile:3]: I think you can install just the certificates by adding [ca-certificates](https://pkgs.alpinelinux.org/package/edge/main/x86/ca-certificates)

```suggestion
RUN apk --no-cache add ca-certificates
```

Comment (drcapulet) [reply Dockerfile:3]: Ah yeah, was trying to build this locally but Docker is acting up and forgot to push this up.

Comment (derektamsen) [reply Dockerfile]: No worries!

**Pull request merged**